docs/basics/0002-installation.md

```diff
@@ -0,0 +1,88 @@
+## How do I install it?
+
+### Step 1.
+Make sure that you have a non-root user account on the control machine.
+
+Let's say the user is named `admin` and the machine
+is named `control.example.com`. Start by logging in as your user via SSH:
+
+```bash
+(laptop)$ ssh admin@control.example.com
+```
+
+(Did that work? Only type the text starting after the `$` in these examples.)
+
+You should see a prompt like
+
+```bash
+admin@control.example.com:~$
+```
+
+Run this command to verify that you are in the home directory of the `commcare-cloud` user.
+
+```bash
+admin@control.example.com:~$ pwd
+/home/commcare-cloud
+```
+
+### Step 2.
+
+Pull commcare-cloud source code.
+
+```bash
+admin@control.example.com:~$ git clone https://github.com/dimagi/commcare-cloud.git
+```
+
+Verify that created a directory called `commcare-cloud`:
+
+```bash
+admin@control.example.com:~$ ls commcare-cloud/
+commcare-cloud-bootstrap  environments  MANIFEST.in   setup.py
+control                   fabfile.py    provisioning  src
+dev_requirements.txt      git-hooks     README.md     tests
+docs                      Makefile      scripts       Vagrantfile
+```
+
+If you see something like
+
+```bash
+ls: cannot access commcare-cloud: No such file or directory
+```
+
+then the `git clone` command did not run correctly
```

Comment:
Add: "Make sure you have git installed and run it again with `--verbose` to give more logging output"?